templates/experiments/single_experiment_home.html

```diff
@@ -389,4 +504,139 @@ <h3 class="font-bold text-lg">Parent Experiments:
 
     window.addEventListener('hashchange', selectTabFromHash);
   </script>
+  <script>
+    const tags = {{ available_tags|safe|default:"[]" }};
+    const versionsList = {{ experiment_versions|safe|default:"[]" }};
+    document.addEventListener('alpine:init', () => {
+      Alpine.data('filterComponent', () => ({
+        filterData: {
+          showFilters: false,
+          filters: [],
+          columns: {
+            'participant': {type: 'string', operators: ['equals', 'contains', 'starts_with', 'ends_with']},
+            'last_message': {type: 'timestamp', operators: ['on', 'before', 'after']},
+            'tags': {type: 'choice', operators: ['any of', 'all of'], options: tags },
+            'versions': {type: 'choice', operators: ['any of', 'all of'], options: versionsList}
+          }
+        },
+        init() {
+          htmx.process(this.$el);
```

Comment:
can you say more about what this line is for

Reply:
I initially included it believing it was necessary to ensure HTMX would recognize/process attributes within the Alpine-rendered components. But I just tested with commenting it out and it works fine. 
I think since the filter functionality triggers HTMX actions via `htmx.ajax()` calls rather than relying on HTMX attributes being processed after Alpine renders elements. Since there's no race condition between Alpine rendering and HTMX processing here, the line is unnecessary